README.md

```diff
@@ -92,15 +92,28 @@ Also see [this presentation](http://tinyclouds.org/jsconf2018.pdf).
 ## Build instructions
 
 To ensure reproducible builds, Deno has most of its dependencies in a git
-submodule. However, you need to install separately:
+submodule. 
+
+#### Prerequistes
 
 1. [Rust](https://www.rust-lang.org/en-US/install.html)
 2. [Node](http://nodejs.org/)
 3. Python 2. [Not 3](https://github.com/denoland/deno/issues/464#issuecomment-411795578).
-4. [ccache](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/ccache) (Optional but helpful for speeding up rebuilds of V8.)
-.
-
-To build:
+4. [ccache](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/ccache) (Optional but helpful for speeding up rebuilds of V8.)..
+5. C/C++ compiler tool chain
+   1.  **Windows**
+       1. Add `python.exe` to `PATH`. E.g. `set PATH=%PATH%;C:\Python27\python.exe`
+       2. [Visual Studio Community 2017](https://www.visualstudio.com/downloads/), make sure to select the option to install all C++ tools and the Windows SDK
+       3. Enable `Debugging Tools for Windows`, Goto Control Panel -> Windows 10 SDK -> Right-Click -> Change -> Change -> Check Debugging Tools for Windows -> Change -> Finish
+       4.  _Note:_ Building via Windows Subsystem for Linux (WSL) is not supported
+   2. **OSX**
+      1. [XCode](https://developer.apple.com/xcode/downloads/) and the Command Line Tools (Xcode -> Preferences -> Downloads), which will install `gcc` and the related toolchain containing `make`
+   3. **Linux**
+      1. `make`
+      2. `gcc`
```

Comment:
We include our own compiler toolchain in `//third_party/v8/third_party/llvm-build/`

I wouldn't mind adding some hints for Windows users if it's really not clear.